### PR TITLE
Created constants for string values used in the code

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Created constants for string values used in the code

### CODE REVIEW

**Comment by sivaschenko** [comment]
@magento combine 930

**Comment by sivaschenko** [comment]
@magento combine 930

**Comment by sivaschenko** [comment]
@magento combine 930

File: AdobeIms/Model/Config.php [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 namespace Magento\AdobeIms\Model;
 
+use Magento\Framework\UrlInterface;
 use Magento\AdobeImsApi\Api\ConfigInterface;
 use Magento\Config\Model\Config\Backend\Admin\Custom;
 use Magento\Framework\App\Config\ScopeConfigInterface;
-use Magento\Framework\UrlInterface;
```
Comment (hws47a) [inline AdobeIms/Model/Config.php]: Please avoid reordering of `use` statements without a reason

File: AdobeIms/Model/Config.php [hunk 2]
```diff
@@ -25,6 +25,7 @@ class Config implements ConfigInterface
     private const XML_PATH_LOGOUT_URL_PATTERN = 'adobe_ims/integration/logout_url';
     private const XML_PATH_DEFAULT_PROFILE_IMAGE = 'adobe_ims/integration/default_profile_image';
     private const XML_PATH_IMAGE_URL_PATTERN = 'adobe_ims/integration/image_url';
+    private const OAUTH_CALLBACK_URL = 'adobe_ims/oauth/callback';
```
Comment (hws47a) [inline AdobeIms/Model/Config.php:28]: Please prefix the constant with `XML_PATH_`

**Review by hws47a — CHANGES_REQUESTED** [review]

File: AdobeIms/Model/Config.php (hunk 2 above)
Comment (sivaschenko) [reply AdobeIms/Model/Config.php:28]: I don't think this prefix is needed. That's an actual URL path, not the configuration path

**Comment by sivaschenko** [comment]
@magento combine 930

**Comment by sivaschenko** [comment]
@magento combine 930

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by sivaschenko** [comment]
@magento combine 930

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@RakeshJesadiya all pull requests have been successfully combined together: 
 - merged https://github.com/magento/adobe-stock-integration/pull/930: Defined Constant.

**Comment by sivaschenko** [comment]
@magento combine 918 919 920 921 922 923 924 925 926 927 928 929 932 939

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
We got some errors while combining pull requests:
 - https://github.com/magento/adobe-stock-integration/pull/939 : Head organization must be the same for target and source pull request

**Comment by sivaschenko** [comment]
@magento combine 918 919 920 921 922 923 924 925 926 927 928 929 932

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@RakeshJesadiya all pull requests have been successfully combined together: 
 - merged https://github.com/magento/adobe-stock-integration/pull/918: Added Constant value
 - merged https://github.com/magento/adobe-stock-integration/pull/919: Added Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/920: Added Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/921: Added Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/922: Added Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/923: Define Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/924: Define Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/925: Define Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/926: Define Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/927: Define Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/928: Use Exception class
 - merged https://github.com/magento/adobe-stock-integration/pull/929: Set Constant Value
 - merged https://github.com/magento/adobe-stock-integration/pull/932: Define Constant of XML path

**Comment by sivaschenko** [comment]
@magento combine 939

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
We got some errors while combining pull requests:
 - https://github.com/magento/adobe-stock-integration/pull/939 : Head organization must be the same for target and source pull request

File: AdobeIms/Block/Adminhtml/SignIn.php [hunk 3]
```diff
@@ -100,7 +104,7 @@ public function getComponentJsonConfig(): string
             )
         );
     }
-
+
```
Comment (sivaschenko) [inline AdobeIms/Block/Adminhtml/SignIn.php]: Please remove unnecessary spaces

File: AdobeStockClient/Model/Client/Files.php [hunk 4]
```diff
@@ -139,7 +147,7 @@ private function getUrl(array $ids, array $columns, string $locale): string
                 ]
             );
     }
-
+
```
Comment (sivaschenko) [inline AdobeStockClient/Model/Client/Files.php]: Please remove unnecessary spaces

File: AdobeStockAdminUi/Block/Adminhtml/System/Config/TestConnection.php [hunk 5]
```diff
@@ -8,17 +8,19 @@
...
-use Magento\Backend\Block\Template\Context;
+use Magento\AdobeStockClientApi\Api\ClientInterface;
 use Magento\Framework\Data\Form\Element\AbstractElement;
 
 /**
  * Adobe Stock test connection block
  */
 class TestConnection extends Field
 {
+    private const XML_PATH_TEST_CONNECTION = 'adobe_stock/system_config/testconnection';
```
Comment (sivaschenko) [inline AdobeStockAdminUi/Block/Adminhtml/System/Config/TestConnection.php]: That is not an xml path, that is an actual url path, please remove the prefix

File: AdobeIms/Model/Config.php (hunk 1 above)
Comment (sivaschenko) [reply AdobeIms/Model/Config.php]: +1 Please revert ordering imports by length. Ordering imports alphabetically is more preferrable

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for your pull requests @RakeshJesadiya . Please take a look at the review comments

Also, please avoid creating similar small PRs in bulk, it's more efficient to review such changes in a single PR.

**Comment by sivaschenko** [comment]
@magento combine 917

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@RakeshJesadiya all pull requests have been successfully combined together: 
 - merged https://github.com/magento/adobe-stock-integration/pull/917: Remove redundant parameter value.

File: AdobeIms/Model/UserProfileRepository.php [hunk 6]
```diff
@@ -91,7 +91,7 @@ public function get(int $entityId): UserProfileInterface
         $entity = $this->entityFactory->create();
         $this->resource->load($entity, $entityId, self::ID);
         if (!$entity->getId()) {
-            $message = __('User profile with id %id not found.', ['id' => $entityId]);
+            $message = __('User profile with id %id not found.', [self::ID => $entityId]);
```
Comment (hws47a) [inline AdobeIms/Model/UserProfileRepository.php]: This is a mistake, array key is applied to '%id' in there string nearby, while ID constant is a column to load data.

File: AdobeIms/Model/UserProfileRepository.php [hunk 7]
```diff
@@ -106,7 +106,7 @@ public function getByUserId(int $userId): UserProfileInterface
         $entity = $this->entityFactory->create();
         $this->resource->load($entity, $userId, self::ADMIN_USER_ID);
         if (!$entity->getId()) {
-            $message = __('User profile with user id %id not found.', ['id' => $userId]);
+            $message = __('User profile with user id %id not found.', [self::ID => $userId]);
```
Comment (hws47a) [inline AdobeIms/Model/UserProfileRepository.php]: This is a mistake, array key is applied to '%id' in there string nearby, while ID constant is a column to load data.

File: AdobeIms/Model/UserProfileRepository.php (hunk 6 above)
Comment (hws47a) [reply AdobeIms/Model/UserProfileRepository.php]: @sivaschenko by the way, I think `ID` constant should not be used in ->load call above. Data interface constant should be used instead.

**Review by hws47a — CHANGES_REQUESTED** [review]

**Comment by RakeshJesadiya** [comment]
@sivaschenko I have made changes. Please Review and check the changes.

**Comment by sivaschenko** [comment]
@RakeshJesadiya Thanks for the changes!

I can actually see that only 2 commits contain changes:

- https://github.com/magento/adobe-stock-integration/pull/931/commits/9fcab0262af9faeba01b0cc0c839f467a60e7be0
- https://github.com/magento/adobe-stock-integration/pull/931/commits/e99d1c40a330c7a645c270bfcd1b02b5b49f1758

Other commits for some reason look empty. Could you please update the use blocks to use alphabetical sorting instead of sorting by length.

**Comment by sivaschenko** [comment]
@RakeshJesadiya would you like to continue progress on this pull request?

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**